src/ledgerCallBuilder.js

```diff
@@ -0,0 +1,20 @@
+import {CallBuilder} from "./callBuilder";
+
+/**
```

Comment:
IMO (i.e. this is just my personal code styling opinion), comments like these as well as the `@constructor` comments are mostly just cruft unless they have descriptive text along with them.  Feel free to leave them out until someone writes that descriptive text.